.eslintignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+template-express-minimal/**
+template-retail-react-app/**
+template-typescript-minimal/**
```

Comment:
These template paths were not matching, since glob pattern like `/**` is relative to the location of the .eslintignore file. It's a subtle difference when compared to passing in just `node_modules`. For more details, see [.gitignore spec](https://git-scm.com/docs/gitignore).

So what would work is: `packages/template-express-minimal/**`. But since we would have more templates in the future, how about this instead?

```suggestion
packages/template-*/**
```

Reply:
Excellent. I was going to get around to do that. Looks like you beat me to it. 

NOTE: I've recently been forced to deal with more globs, [this](https://www.digitalocean.com/community/tools/glob?comments=true&glob=packages%2Ftemplate-%2A%2F%2A%2A&matches=false&tests=packages%2Ftemplate-express-minimal%2Ftest%2Fblah.ts&tests=packages%2Ftemplate-retail-react-app%2Ffile.js&tests=packages%2Ftemplate-typescript-minimal) tool helps when testing.